modules/roles_profiles/manifests/profiles/mac_v3_signing.pp

```diff
@@ -30,6 +30,9 @@
             $role = $facts['networking']['hostname']? {
                 /^dep-mac-v3-signing\d+/ => 'dep',
                 /^tb-mac-v3-signing\d+/ => 'tb-prod',
+                # TODO: Remove 18 (dev machine)
+                # Hard-coded VPN hosts
+                /^mac-v3-signing(18|19|20)$/ => 'vpn',
```

Comment:
- [ ] Ask relops to change these workers' hostnames
- [ ] Update regex to match new hostnames